docs/source/reference/kubernetes/index.rst

```diff
@@ -105,3 +105,4 @@ Table of Contents
    kubernetes-troubleshooting
    multi-kubernetes
    SkyPilot vs. Vanilla Kubernetes <skypilot-and-vanilla-k8s>
+   Examples <examples/index>
```

Comment:
"Examples" here feels odd, and the current set of examples don't necessarily benefit a large set of SkyPilot k8s users. To reduce ToC clutter, move to under `kubernetes-setup`?

Reply:
"Examples" likely attracts more people than setup subpages. One analogy: https://docs.ray.io/en/latest/train/examples.html.